site/homepage/layouts/partials/head.html

```diff
@@ -3,16 +3,16 @@
   <meta name="robots" content="all,follow">
   <meta name="googlebot" content="index,follow,snippet,archive">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  {{ $title_plain := .Title | markdownify | plainify }}
-  <title>{{ $title_plain }} :: {{ .Site.Title | safeHTML }}</title>
+  {{ $title_plain := .Title | plainify | markdownify }}
+  <title>{{ $title_plain }}</title>
```

Comment:
Does removing the site title (i.e. Eclipse Hono) help SEO? Or did you remove it for esthetical reasons? 
At least I understand [Mozilla's explanation](https://moz.com/learn/seo/title-tag) as a recommendation to add the "brand" at the end of the title.

Reply:
I mainly removed it in order to reduce the differences to the original template.
The feedback we got from marketing was that it is most important to have a meaningful and different title for each page. And that the title length should not exceed ~60 characters because everything after that is usually cut off when displaying the title. Based on that, I do not think that adding the site title at the end does any harm. However, I have to admit that I do not really understand the meaning of the explanation from the document you referred to:

> 5. Take advantage of your brand
>
> If you have a strong, well-known brand, then adding it to your titles may help boost click-through rates. We generally still recommend putting your brand at the end of the title, but there are cases (such as your home page or about page) where you may want to be more brand-focused. As mentioned earlier, Google may also append your brand automatically to your display titles, so be mindful of how your search results are currently displayed.

Reply:
Throughout the document, the examples contain the "brand". The recommended format contains it at the end, without saying it is optional.

> ###Optimal format
>
>Primary Keyword - Secondary Keyword | Brand Name
8-foot Green Widgets - Widgets & Tools | Widget World

What I got:
1. The most important and unique keywords should be at the beginning
2. The preview is cut off visually after 600px (~50-60 chars) in the preview box
3. The content of the title is relevant for the ranking, therefore having the site title at the end _might_ still affect the ranking 

> I do not think that adding the site title at the end does any harm

Me too. It might improve the ranking regardless of not being displayed in the preview. So, I would prefer to add it.

Reply:
Ok, I have appended the site title to the page title again ...